sdk/identity/identity/Troubleshooting.md

```diff
@@ -1,48 +1,85 @@
+
+### AuthenticationError
+
+The `AuthenticationError` is used to indicate about a failure to authenticate with Azure Active Directory. The `errorResponse` field contains more details about the specific failure.
+
+## Finding Relevant Information in Error Messages
 
-Follow the configuration instructions in the "Credential unavailable error" sections of the troubleshooting guidelines below for the respective credential/authentication type you're looking to use via `DefaultAzureCredential`:
+`AuthenticationRequiredError` is thrown when unexpected errors occurred while a credential is authenticating. This can include errors received from requests to the AAD STS and often contains information helpful to diagnosis. Consider the following `AuthenticationRequiredError` message.
 
-| Credential Type               | Troubleshooting Guide                                                                      |
-| ----------------------------- | ------------------------------------------------------------------------------------------ |
-| Environment credential        | [Environment credential guide](#troubleshoot-environment-credential-authentication-issues) |
-| Managed Identity credential   | [Managed Identity guide](#troubleshoot-managed-identity-authentication-issues)             |
-| Visual Studio Code credential | [Visual Studio Code guide](#troubleshoot-visual-studio-code-authentication-issues)         |
-| Azure CLI credential          | [Azure CLI guide](#troubleshoot-azure-cli-authentication-issues)                           |
-| Azure PowerShell credential   | [Azure PowerShell guide](#troubleshoot-azure-powershell-authentication-issues)             |
+<--TODO-->
+This error contains several pieces of information:
 
-## Logging
+- **Failing Credential Type**: The type of credential that failed to authenticate. This can be helpful when diagnosing issues with chained credential types such as `DefaultAzureCredential` or `ChainedTokenCredential`.
+
+- **STS Error Code and Message**: The error code and message returned from the Azure AD STS. This can give insight into the specific reason the request failed. For instance in this specific case because the provided client secret is incorrect. More information on STS error codes can be found [here](https://docs.microsoft.com//azure/active-directory/develop/reference-aadsts-error-codes#aadsts-error-codes).
+
+- **Correlation ID and Timestamp**: The correlation ID and call Timestamp used to identify the request in server-side logs. This information can be useful to support engineers when diagnosing unexpected STS failures.
+
+### Enabling and Configuring Logging
+
+The Azure Identity library provides the same [logging capabilities](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Diagnostics.md#logging) as the rest of the Azure SDK.
```

Comment:
```suggestion
The Azure Identity library has the same [logging capabilities](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Diagnostics.md#logging) as the rest of the Azure SDK.
```
nit: Provide feels like if the Identity library provided logging to other SDK clients. I know it doesn’t say that literally, but I have the feeling it could be misinterpreted.